packages/protocol/contracts/governance/Validators.sol

```diff
@@ -706,19 +749,39 @@ contract Validators is
     return balance >= getAccountLockedGoldRequirement(account);
   }
 
+  /**
+   * @notice Returns the validator BLS key.
+   * @param signer The account that registered the validator or its authorized signing address.
+   * @return The validator BLS key.
+   */
+  function getValidatorBlsKeyFromSigner(address signer)
+    external
+    view
+    returns (bytes memory blsKey)
+  {
+    address account = getAccounts().validatorSignerToAccount(signer);
```

Comment:
Is there a reason this is `validatorSignerToAccount` instead of `activeValidatorSignerToAccount`?

Reply:
Hmm, I'm not certain it matters in this case but I feel that `validatorSignerToAccount` here is safer. Wdyt?

Reply:
I don't think it is safer, but on second look I aggree that is generally doesn't matter. One thing is though that it might be assumed that the input needs to be the validators authorized signer for validation, but the result would be the same if the input signer was authorized for voting or attestation instead.